### PR TITLE
Simplify introduction, licensing. Explain how to contribute to docs

### PR DESCRIPTION
**Pull request by hamishwillee**
This makes significant improvement (if I do say so myself) to the introduction and contribution sections. It better highlight support channels and what you can do with the DevGuide. It also adds a missing license file in the repo itself.

### CODE REVIEW

File: en/README.md [hunk 1]
```diff
@@ -2,27 +2,53 @@
 
 [![Releases](https://img.shields.io/github/release/PX4/Firmware.svg)](https://github.com/PX4/Firmware/releases) [![Discuss](https://img.shields.io/badge/discuss-px4-ff69b4.svg)](http://discuss.px4.io/) [![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io) 
 
-> **Info** Developers only! This guide is under active development and not intended for consumers.
+> **Info** This guide is for primarily for software developers and (new) hardware integrators.
+> To fly, build and modify vehicles using supported hardware see the [PX4 User Guide](https://docs.px4.io/en/).
 
-This guide describes how to work inside the PX4 system architecture. It enables developers to:
+This guide explains how to:
```
Comment (hamishwillee) [inline en/README.md:8]: @LorenzMeier @bkueng This update to the home page is intended to provide better explanation of what you can do with the DevGuide. Some of it is "aspirational" - currently we don't really explain how you can "add new flight modes", but this is where that information should go.

File: en/contribute/docs.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+1. Make the desired change.
+1. At the bottom of the page you'll be prompted to create a separate branch and then 
+   guided to submit a *pull request*.
+   
+The documentation team reviews submitted pull requests and will either merge
+it or work with you to update it.
+
+## Adding New Content - Big Changes
+
+### What Goes Where?
```
Comment (hamishwillee) [inline en/contribute/docs.md:25]: @LorenzMeier This addresses your question "could we get these instructions into the dev guide so we can point people who want to write articles in the future to it?"
- from https://github.com/PX4/Devguide/pull/348#issuecomment-344853767

File: en/README.md [hunk 3]
```diff
@@ -2,27 +2,53 @@
...
+* Communicate/integrate with external robotics APIs.
+
+
+## Forums and Chat {#support}
+
+The core development team and community are active on the following forums and chat channels.
+
+* [PX4 Discuss](http://discuss.px4.io/) (*recommended*)
+* [Slack](http://slack.px4.io) (sign up)
+* [Gitter](https://gitter.im/PX4/Firmware)
```
Comment (bkueng) [inline en/README.md]: Didn't we deprecate gitter?

File: en/contribute/docs.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
+# Contributing to Documentation
+
+Contributions to the DroneCore guides, including the PX developer and user guides are very welcome!
```
Comment (bkueng) [inline en/contribute/docs.md]: DroneCode?
typo: `PX` -> `PX4`

File: en/contribute/docs.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+* A web hook is used to track whenever files are merged into the master branch on this repository, causing the book to rebuild.
+
+
+
+## Style guide
+
+1. Files/file names
+
+   * Put new files in an appropriate sub-folder
+   * Use descriptive names. In particular, images should describe what the contain.
```
Comment (bkueng) [inline en/contribute/docs.md]: *they*

File: en/contribute/docs.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+* The _structure_ of the book is defined in a file named **SUMMARY.md**.
+* This is a [multilingual](https://toolchain.gitbook.com/languages.html) book, 
+  so there is a **LANGS.md** file in the root directory defining what languages are supported. 
+  Pages for each language are stored in the folder named for the associated language code \(e.g. "zh" for Chinese, "en" for English\). 
+* A file named **book.json** defines any dependencies of the build.
+* A web hook is used to track whenever files are merged into the master branch on this repository, causing the book to rebuild.
+
+
+
+## Style guide
```
Comment (bkueng) [inline en/contribute/docs.md:61]: 👍

**Review by bkueng — COMMENTED** [review]
I noticed a few things, but generally looks good.

File: en/README.md (hunk 3 above)
Comment (hamishwillee) [reply en/README.md]: Yes. Good catch. I will fix.

**Comment by hamishwillee** [comment]
Thanks very much for review @bkueng ! All fixed. Merging now.

**Pull request merged**